packages/graph-explorer-proxy-server/node-server.js

```diff
@@ -108,7 +108,7 @@ dotenv.config({ path: "../graph-explorer/.env" });
       );
       
       if (!response.ok){
-        let msg = `Error getting response the sparql endpoint. [${response.statusText}] with query [${req.query.query}].`
+        let msg = `Error getting response the SPARQL Server endpoint. [${response.statusText}] with query [${req.query.query}].`
```

Comment:
nit: just SPARQL endpoint. There is no SPARQL Server.

Reply:
Thanks for pointing.